frontend/src/services/transactions/request-manager.ts

```diff
@@ -68,12 +57,6 @@ export async function sendRequestTransaction(
 
   const transactionReceipt = await transaction.wait();
   request.receipt = transactionReceipt;
-  //events is added dynamically when wait method called!
-  // TODO verification of other args?
-  const event = findFirstEvent(request.receipt, 'RequestCreated');
-  if (!event) {
-    //TODO error
-  }
-  request.requestId = event.args.requestId;
```

Comment:
This code has been left from the old implementation.. currently, it's useless because we use another way of detecting fulfillment by listening to the events on the target chain directly. This code was from the old concept of waiting user to change between chains, so no longer needed!